bin/wsdump.py

```diff
@@ -63,7 +68,9 @@ def parse_args():
 
     return parser.parse_args()
 
-class RawInput():
+
+class RawInput:
```

Comment:
Yeah, it should be, but I wasn't sure if that would have some adverse impact downstream, and the test coverage isn't very high. If you don't think there would be any adverse impact, I'll change it.